lib/driver.js

```diff
@@ -562,10 +562,13 @@ class XCUITestDriver extends BaseDriver {
         await markSystemFilesForCleanup(this.wda);
       }
 
-      // TODO: platformVersion should be a required cap
-      if (this.isSafari() && util.compareVersions(this.opts.platformVersion, '>=', '12.2')) {
-        // on 12.2 the page is not opened in WDA
-        await this.setUrl(this._currentUrl);
```

Comment:
should we keep this for real devices?

Reply:
It can't happen here. Until we're in a web context this will just open Siri on a real device.

Reply:
can we then set the url using normal xctest primitives (aka type it in the address bar and click Go?)